src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -292,6 +292,18 @@ public WorkspaceDescription updateWorkspace(
         "Error updating workspace");
   }
 
+  /**
+   * Call the Workspace Manager POST "/api/workspaces/v1/{id}/gcp/enablepet" endpoint to grant a
+   * user permission to impersonate their own pet service account in a workspace.
```

Comment:
nit 1: to grant a user and their pet this permission
nit 2: "the currently logged in user" instead of "a user"

[dr seuss rhyme here]